app/assets/stylesheets/app/dashboard.scss

```diff
@@ -187,6 +189,15 @@
     justify-content: space-evenly;
     color: $primary;
 
+    &--fullscreen {
+      margin: 0;
+      width: 100%;
+
+      .matrix__values {
```

Comment:
Selector `matrix__values` should be written in lowercase with hyphens

Reply:
Alega por cosas de BEM